buildkite/src/Jobs/BlockProductionTest.dhall

```diff
@@ -0,0 +1,35 @@
+let Pipeline = ../Pipeline/Dsl.dhall
+let JobSpec = ../Pipeline/JobSpec.dhall
+let Command = ../Command/Base.dhall
+let OpamInit = ../Command/Base.dhall
+let Docker = ../Command/Docker/Type.dhall
+let Size =  ../Command/Size.dhall
+let Cmd = ../Lib/Cmds.dhall
+let S  = ../Lib/SelectFiles.dhall
+let DockerArtifact = ../Command/DockerArtifact.dhall
+let OpamInit = ../Command/OpamInit.dhall
+in
+let name = "BlockProductionTest"
+let runTestExecutive =
+  Command.build
+    Command.Config::
+      { commands = OpamInit.andThenRunInDocker (["BUILDKITE_GS_APPLICATION_CREDENTIALS_JSON"]) "bash buildkite/scripts/run-test.sh"
+      , label = "Build and run test-executive"
+      , key = "run-test-executive"
+      , target = Size.Large
+      , docker = None Docker.Type
+      , depends_on = [ { name = "Artifact", key = "artifacts-build" } ]
```

Comment:
just to double-check: is the dependency for the test on the daemon deb package or Docker image?

Reply:
Good catch. I changed the dependency to point at "docker-artifact" instead.